### DISCUSSION TITLE
Writing the Results of Locust Run to External DB like BigQuery

### DISCUSSION THREAD

**Discussion by sandeep540** [locustio/locust#2845] [Q&A]
How can I write the below 4 results to External DB like BigQuery for BI Reporting, any examples?

1. report_api_test_exceptions
2. report_api_test_failures
3. report_api_test_stats
4. report_api_test_stats_history

**Comment by cyberw** [comment]
I’ve never done that specifically. But you can register a listener for maybe the test_stop event and read from the stats object.
